variants/DISCO_B_G431B_ESC1/PeripheralPins.c

```diff
@@ -150,7 +150,7 @@ WEAK const PinMap PinMap_PWM[] = {
   //  {PB_9,  TIM17,  STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM17, 1, 0)}, // TIM17_CH1
   //  {PB_10, TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 3, 0)}, // TIM2_CH3
   //  {PB_11, TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 4, 0)}, // TIM2_CH4
-  {PB_13, TIM1,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF6_TIM1, 1, 1)}, // TIM1_CH1N
+  //  {PB_13, TIM1,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF6_TIM1, 1, 1)}, // TIM1_CH1N
```

Comment:
Why this pin is incorrect ?
![image](https://user-images.githubusercontent.com/20641798/98921213-4f2c4c00-24d1-11eb-8894-547f79c92c97.png)

Reply:
I think you are looking at a generic g431b pin usage - this board has been wired up with a specific purpose.  Page 15-16 of [b-g431b-esc1 user guide](https://www.st.com/resource/en/user_manual/dm00564746-electronic-speed-controller-discovery-kit-for-drones-with-stm32g431cb-stmicroelectronics.pdf
) explains why this change is correct.
pb13 is not connected on this discovery kit:
![image](https://user-images.githubusercontent.com/15650427/99254128-c4698b00-2809-11eb-826f-9b348bcd9620.png)

...and TIM1_CH1N is associated with PC13 on this board (it is connected to the negative drive gate of phase A motor driver)

![image](https://user-images.githubusercontent.com/15650427/99253930-794f7800-2809-11eb-84a5-9ea4544b7af3.png)